src/smc-webapp/project.coffee

```diff
@@ -116,11 +116,16 @@ class ProjectPage
         delete @actions
         delete @store
 
+    get_total_project_quotas: (s, project_id) =>
+        quotas = s.get_total_project_quotas(project_id)
+        quotas.project_id = project_id
```

Comment:
right here, I got `project.coffee?a24e:121 Uncaught TypeError: Cannot set property 'project_id' of undefined`

so, it looks as if `s.get_total_project_quotas` sometimes returns null or undefined?